app/javascript/packs/actions/buildings.js

```diff
@@ -81,26 +107,65 @@ export async function updateBuilding(id, updatedBuilding, dispatch) {
       ...FETCH_SETTINGS
     }).then(resp => resp.json());
 
-    dispatch(saveResponse(UPDATE_BUILDING, id, response.data));
+    dispatch(saveBuilding(UPDATE_BUILDING, id, response.data));
   } catch (error) {
     dispatch(saveError(UPDATE_BUILDING, id, error));
   };
 }
 
-function saveResponse(type, id, response) {
+  /////////////////////////////////////////////////////////////////
```

Comment:
You can actually have the answer actions exported in their own answers file in actions folder.